CHANGELOG.md

```diff
@@ -11,7 +17,7 @@
 
 ## 1.4.2
 
-- Support POST method in `CurlClient`. ([#9](https://github.com/isoos/http_client/pull/9) by [rbandres98](https://github.com/rbandres98))
+- Support POST method in `CurlClient`. ([#9](https://github.com/isoos/http_client/pull/9) by [andresruizdev](https://github.com/andresruizdev))
```

Comment:
I think you wanted to copy this block up to 1.5.2, but instead edited it in-place.

Reply:
> I think you wanted to copy this block up to 1.5.2, but instead edited it in-place.

rbandres98 is my old username, and currently CHANGELOG.md is redirecting to the profile https://github.com/rbandres98, a profile that no longer exists since I updated my username to andresruizdev (https://github.com/andresruizdev). So I just updated the changelog with the newer username and url.

Reply:
Oh, alright. Still, please add the same note to the 1.5.2 entry.

Reply:
I forgot to upload the pull request reference in the changelog. I already added it too, sorry.